### ISSUE TITLE
[a11y-defect-2]: 2 issues related to headings on Benefits and Claims, Dependents Form Audits: Forms 21-686c and 21-674

### ISSUE THREAD

**Issue by coforma-jamie** [department-of-veterans-affairs/va.gov-team#63206]
### Point of contact

Jamie Klenetsky Fay

### Severity level

2, Serious. Should be fixed in 1-2 sprints post-launch.

### Details

These are issues encountered during an accessibility audit of #61685.  

HTML should have properly nested headings. I found improperly nested headings in 2 locations:

- [Step 7: Remove a stepchild / stepchild status](https://staging.va.gov/view-change-dependents/add-remove-form-21-686c/686-stepchild-no-longer-part-of-household/0): "Step 7 of 12..." is H2, and the next heading is H4. 
![42009739-c4976523ac83f1c0dba8c0f20f1718b4](https://github.com/department-of-veterans-affairs/va.gov-team/assets/135633989/57783f60-b12e-4c18-ba1c-bdf364daac30)
- [Step 12: Review and submit](https://staging.va.gov/view-change-dependents/add-remove-form-21-686c/review-and-submit): There are many repeated headings (headings with the exact same text) on this page. This is confusing to AT users who navigate a page by heading - they won't know what's contained within those sections. I've highlighted the duplicate headings in [this screenshot](https://www.awesomescreenshot.com/image/42006588?key=1c28a83d6fbba74908fa66736e005a3e) (scroll down - the duplications start at "Information about your spouse's former marriage(s)).

### Reproduction steps

1. Log onto staging (I used an ID.me test user)
2. Go to [the change/remove dependents form](https://staging.va.gov/view-change-dependents/)
3. Start filling out the form. On the options screen, select every checkbox.
4. You'll see the errors on the "stepchild status" and "review and submit" screens. You can use the [Chrome HeadingsMap extension](https://chrome.google.com/webstore/detail/headingsmap/flbjommegcjonpdmenkdiocclhjacmbi) to view the heading levels on the "stepchild status" page.

### Proposed solution or next steps

- [ ]  For the "stepchild status" page error, the stepchild's name - which is currently an H4 - should be an H3.

- [ ] For the "review and submit" page errors, you'll need make every header unique. This is a content fix. You may do something like this:
- "Information needed to add your child: Name and Date of Birth" / "Information needed to add your child: Status" / "Information needed to add your child: Location" OR
- "Your child's name and date of birth" / "Your child's status" / "Your child's location" 
- or similar.

### Acceptance Criteria
- [ ] The stepchild's name on the "stepchild status" page is an H3 instead of H4
- [ ] All headings on the "review and submit page" are unique

### References, articles, or WCAG support

1. [Success Criterion 2.4.6: Headings and Labels](https://www.w3.org/WAI/WCAG21/Understanding/headings-and-labels)
2. [Providing descriptive headings](https://www.w3.org/WAI/WCAG21/Techniques/general/G130)
3. [Heading ranks](https://www.w3.org/WAI/tutorials/page-structure/headings/#heading-ranks)

### Type of issue

- [X] Screenreader
- [X] Keyboard
- [ ] Focus
- [ ] Headings
- [ ] Color
- [ ] Zoom
- [X] Semantics
- [ ] Axe-core
- [ ] Something else

**Comment by coforma-jamie** [comment]
Not a duplicate, I found different heading-related issues than #30026. Thanks for pointing out that one though!
